### PR TITLE
Create ES mapping when starting indexing.

### LINKED ISSUE #385: Define static prefix mappings for metadata fields in default ES view
The [static mapping](https://github.com/BlueBrain/nexus-iam/blob/master/modules/elastic/src/main/resources/elastic-mappings.json) for the key values of the metadata has to be created and the index itself has to be created with that mapping on [ElasticIndexer](https://github.com/BlueBrain/nexus-kg/blob/master/src/main/scala/ch/epfl/bluebrain/nexus/kg/indexing/ElasticIndexer.scala#L95)

### PR DESCRIPTION
**Pull request by wwajerowicz**
Fixes #385

### CODE REVIEW

File: src/main/resources/elastic/mapping.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "mappings": {
+    "{{docType}}": {
+      "properties": {
+        "@type": {"type": "keyword"},
+        "@id": {"type": "keyword"},
+        "_rev": {"type": "keyword"},
```
Comment (umbreak) [inline src/main/resources/elastic/mapping.json]:
```json
"_rev": {"type": "long"}
```

File: src/main/resources/elastic/mapping.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+        "_deprecated": {"type": "boolean"},
+        "_createdAt": {"type": "date"},
+        "_updatedAt": {"type": "date"},
+        "_createdBy": {"type": "keyword"},
+        "_updatedBy": {"type": "keyword"},
+        "_constrainedBy": {"type": "keyword"},
+        "_source": {"type": "text"}
```
Comment (umbreak) [inline src/main/resources/elastic/mapping.json]: What about the `_original` field ?

Comment (wwajerowicz) [reply src/main/resources/elastic/mapping.json]: I renamed `original` to `_source` as I thought that would be more meaningful.

Comment (umbreak) [reply src/main/resources/elastic/mapping.json]: But it isn't renamed right...it is still [here](https://github.com/wwajerowicz/nexus-kg/blob/a366ebb21a86dc131bf3ec323d8449b79b2e60cb/src/main/scala/ch/epfl/bluebrain/nexus/kg/indexing/ElasticIndexer.scala#L71)

Comment (wwajerowicz) [reply src/main/resources/elastic/mapping.json]: Yes, I thought that I renamed it, but I only added `_` 😄

**Pull request merged**